AEPServices/Mocks/AEPTestUtils/MockHitProcessor.swift

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright 2021 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+@testable import AEPServices
```

Comment:
I am confused with all these new mock and test utils files, shouldn't we use the testutils dependency instead? - e.g. https://github.com/adobe/aepsdk-edge-ios/blob/staging/Podfile#L37

Reply:
We ran into an issue using the TestUtil pod because it caused a circular dependency with Core. We are going to look into it post release.